src/perception/se3.hpp

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <string>
+
+#include <tf2_ros/buffer.h>
+#include <tf2_ros/transform_broadcaster.h>
+
+#include <geometry_msgs/Pose.h>
+
+class SE3 : public geometry_msgs::Pose {
```

Comment:
I would honestly rather not inherit from Pose and just have it own an Eigen::Vector3d for position and an Eigen::Matrix<double, 4, 1> for its quat. I think thats what @rbridges12 is doing anyway

Reply:
True, we need to agree on internal representation so that we can have it consistent between Python and C++

Reply:
as per our discussion, we should use a vector3d for position/translation and a separate rotation class for rotation/orientation right?

Reply:
I ended up using an `Eigen::Vector3d` and `Eigen::Quaterniond`